app/models/book_suggestion.rb

```diff
@@ -0,0 +1,4 @@
+class BookSuggestion < ApplicationRecord
+  validates :author, :link, :title, :editor, :year, presence: true
+  belongs_to :user
```

Comment:
You do not need an `optional: true` here?

Reply:
yes sir!